<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Vojta Molda</author>
 <class>QBoardWindow</class>
 <widget class="QMainWindow" name="QBoardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USB Measuring Board</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resrc/board.qrc">
    <normaloff>:/icon/board.png</normaloff>:/icon/board.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QBoardWidget" name="boardWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <property name="icon">
      <iconset resource="../resrc/board.qrc">
       <normaloff>:/icon/choose-language.png</normaloff>:/icon/choose-language.png</iconset>
     </property>
     <addaction name="actionDefault"/>
     <addaction name="separator"/>
     <addaction name="actionEnglish"/>
     <addaction name="actionFrench"/>
     <addaction name="actionCzech"/>
    </widget>
    <addaction name="menuLanguage"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuDevice"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
  </widget>
  <widget class="QToolBar" name="deviceToolBar">
   <property name="windowTitle">
    <string>Device Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/board.png</normaloff>:/icon/board.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/configure.png</normaloff>:/icon/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/edit-cut.png</normaloff>:/icon/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/edit-copy.png</normaloff>:/icon/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/edit-paste.png</normaloff>:/icon/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/edit-redo.png</normaloff>:/icon/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/dialog-cancel.png</normaloff>:/icon/dialog-cancel.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionCzech">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/language/cs.png</normaloff>:/language/cs.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Czech</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/language/fr.png</normaloff>:/language/fr.png</iconset>
   </property>
   <property name="text">
    <string>&amp;French</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/language/en.png</normaloff>:/language/en.png</iconset>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="icon">
    <iconset resource="../resrc/board.qrc">
     <normaloff>:/icon/edit-undo.png</normaloff>:/icon/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Default</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QBoardWidget</class>
   <extends>QWidget</extends>
   <header location="global">boardwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resrc/board.qrc"/>
 </resources>
 <connections/>
</ui>
